templates/search/blocks/search_results.html

```diff
@@ -1,6 +1,12 @@
 {% load search_tags %}
 {% load humanize %}
 
+<script>
+    //var $noOfFilters = 0;
+    var $noOfFilters = {{ selected_filters_count }};
```

Comment:
`var $noOfFilters = {{ selected_filters_count }};` 

- Could this be security vulnerable? 
- Could it be safer to use [json_script-v4.0](https://docs.djangoproject.com/en/4.0/ref/templates/builtins/#json-script) template tag and pick up the data exclusively with JSON.parse?
- Would this article [Safely Including Data for JavaScript in a Django Template](https://adamj.eu/tech/2020/02/18/safely-including-data-for-javascript-in-a-django-template/#the-secure-way) help?

Reply:
On checking the View Page Source it appears that `{{ selected_filters_count }}` is evaluated.

Reply:
I have removed the unused code @TNA-Allan 
I don't think var $noOfFilters = {{ selected_filters_count }}; would be as security issue. Using the json_script-v4.o tag would take quite a bit of development I think

Reply:
> Using the json_script-v4.o tag would take quite a bit of development I think.

Yes, small bit.  Would this help?
```
{{ selected_filters_count|json_script:"selected_filters_count" }}
<script>    
    $noOfFilters = JSON.parse(document.getElementById('selected_filters_count').textContent);
    let $buttonHtml= "Filters <span class='filter-indicator'>"+ $noOfFilters +"</span>";
</script>
```
> I don't think var $noOfFilters = {{ selected_filters_count }}; would be as security issue.

I'm not so sure :) how to validate if its a security issue at the moment , but I wanted to point it out with those links I found as it seems obvious.  For the same reason, I will leave it to you - _check and update the code above_, otherwise mark as _resolved_

Reply:
Thanks for that @TNA-Allan , I've updated the code now and will mark this as resolved